peer/brontide.go

```diff
@@ -1201,6 +1201,7 @@ func (p *Brontide) addLink(chanPoint *wire.OutPoint,
 		GetAliases:              p.cfg.GetAliases,
 		PreviouslySentShutdown:  shutdownMsg,
 		DisallowRouteBlinding:   p.cfg.DisallowRouteBlinding,
+		DisallowQuiescence:      false, // TODO(proofofkeags): fix
```

Comment:
Shouldn't this be true and we can just enable it for itests in the meantime?

Reply:
We'd need a method of enabling it in the itests. I haven't investigated this angle. I'll take a look tomorrow. It's late here haha.

Reply:
We can use [ProtocolOptions](https://github.com/lightningnetwork/lnd/blob/master/lncfg/protocol_integration.go#L17) to add an `EnableQuiescence` option and enable it only for itests.  Did this for [route blinding](https://github.com/lightningnetwork/lnd/pull/8160/commits/6e5eead61721dd647074d2f069beee3f8e97dc8b) to be able to test things before we were ready to ship.